services/sddm-tmpfiles.conf.in

```diff
@@ -0,0 +1,8 @@
+# Home dir of the sddm user, also contains state.conf
+d	${STATE_DIR}	0750	sddm	sddm
+# This contains X11 auth files passed to Xorg and the greeter
+d	${RUNTIME_DIR}	0711	root	root
```

Comment:
Not necessarily in scope of this PR but `RUNTIME_DIR` should probably be migrated from `/var/run` to `/run`. >=cmake-3.9 would have `CMAKE_INSTALL_RUNSTATEDIR` for that purpose which could then be set by distros accordingly.

Reply:
Yep, currently `-DRUNTIME_DIR=/run` is necessary to get that.